Test/Unit/Helper/CartTest.php

```diff
@@ -2694,74 +2695,95 @@ public function getCartData_withEmptyImmutableQuote_returnsEmptyArray()
         */
         public function getCartData_whenPaymentOnlyAndHasOrderPayload_returnsCartData()
         {
-        $currentMock = $this->getCurrentMock(
-            [
-                'setLastImmutableQuote',
-                'getCartItems',
-                'getQuoteById',
-                'collectDiscounts',
-                'createImmutableQuote',
-                'getCalculationAddress'
-            ]
+        $this->skipTestInUnitTestsFlow();
+        $boltHelperCart = Bootstrap::getObjectManager()->create(BoltHelperCart::class);
+        $quote = Bootstrap::getObjectManager()->create(Quote::class);
+        $quote->setQuoteCurrencyCode("USD");
+
+        $product = TestUtils::createSimpleProduct();
+        $this->objectsToClean[] = $product;
+        $quote->addProduct($product,1);
+        $quote->getShippingAddress()->setFirstname($this->testAddressData['first_name']);
```

Comment:
Can we have a helper for setting the address to quote?

Reply:
@vitaliyreznikov Yes. I've updated the code. Please re-check